src/i18n/fi.json

```diff
@@ -129,6 +130,8 @@
     "validation-stringLengthCounter": " merkkiä jäljellä",
     "validation-requiredInContentLanguages": "Tämä kenttä vaaditaan kaikilla valituilla kielillä",
     "validation-offerIsFreeOrHasPrice": "Maksullinen tapahtuma vaatii hinnan kaikilla valituilla kielillä",
+    "validation-atLeastOneIsTrue": "Valitse vähintään yksi toistopäivä",
+    "validation-isMoreThanOne": "Toistovälin tulee olla vähintään 1 viikko",
```

Comment:
Ah, now I see. The isMoreThanOne message will be visible when "Toistoväli" is set to zero, so it works as intended :)

What I was doing is selecting days in which no single repeating event occurs, such as repeating every Monday and selecting the start and end dates so that no Monday is included. Don't know if we should add a validation for such a case as well?